<!DOCTYPE html>
<html lang="es" xml:lang="es">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Referencias | Caracterización morfológica, fisiológica y molecular de entradas de algodón (Gossypium hirsutum L.) e identificación de QTL de importancia agronómica</title>
  <meta name="description" content="Referencias | Caracterización morfológica, fisiológica y molecular de entradas de algodón (Gossypium hirsutum L.) e identificación de QTL de importancia agronómica" />
  <meta name="generator" content="bookdown 0.39.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Referencias | Caracterización morfológica, fisiológica y molecular de entradas de algodón (Gossypium hirsutum L.) e identificación de QTL de importancia agronómica" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Referencias | Caracterización morfológica, fisiológica y molecular de entradas de algodón (Gossypium hirsutum L.) e identificación de QTL de importancia agronómica" />
  
  
  

<meta name="author" content="Ing. Agr. Pablo Nahuel Dileo" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="B-segundo-apéndice.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Contenido preliminar</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#agradecimientos"><i class="fa fa-check"></i>Agradecimientos</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#dedicatoria"><i class="fa fa-check"></i>Dedicatoria</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publicaciones"><i class="fa fa-check"></i>Publicaciones</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#resumen"><i class="fa fa-check"></i>Resumen</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="introducción.html"><a href="introducción.html"><i class="fa fa-check"></i>Introducción</a></li>
<li class="chapter" data-level="1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html"><i class="fa fa-check"></i><b>1</b> Caracterización morfológica</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#introducción-1"><i class="fa fa-check"></i><b>1.1</b> Introducción</a></li>
<li class="chapter" data-level="1.2" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#objetivo"><i class="fa fa-check"></i><b>1.2</b> Objetivo</a></li>
<li class="chapter" data-level="1.3" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#materiales-y-métodos"><i class="fa fa-check"></i><b>1.3</b> Materiales y métodos</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#variables-medidas"><i class="fa fa-check"></i><b>1.3.1</b> Variables medidas:</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#análisis-estadístico"><i class="fa fa-check"></i><b>1.3.2</b> Análisis estadístico</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#resultados"><i class="fa fa-check"></i><b>1.4</b> Resultados</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#fenología"><i class="fa fa-check"></i><b>1.4.1</b> Fenología</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#mapeo-y-descriptores-morfológicos"><i class="fa fa-check"></i><b>1.4.2</b> Mapeo y descriptores morfológicos</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#componentes-del-rendimiento-y-calidad-de-fibra"><i class="fa fa-check"></i><b>1.4.3</b> Componentes del rendimiento y calidad de fibra</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#discusión"><i class="fa fa-check"></i><b>1.5</b> Discusión</a></li>
<li class="chapter" data-level="1.6" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#conclusión"><i class="fa fa-check"></i><b>1.6</b> Conclusión</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-math-sci.html"><a href="2-math-sci.html"><i class="fa fa-check"></i><b>2</b> Caracterización fisiológica</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-math-sci.html"><a href="2-math-sci.html#introducción-2"><i class="fa fa-check"></i><b>2.1</b> Introducción</a></li>
<li class="chapter" data-level="2.2" data-path="2-math-sci.html"><a href="2-math-sci.html#objetivo-1"><i class="fa fa-check"></i><b>2.2</b> Objetivo</a></li>
<li class="chapter" data-level="2.3" data-path="2-math-sci.html"><a href="2-math-sci.html#materiales-y-métodos-1"><i class="fa fa-check"></i><b>2.3</b> Materiales y métodos</a></li>
<li class="chapter" data-level="2.4" data-path="2-math-sci.html"><a href="2-math-sci.html#resultados-1"><i class="fa fa-check"></i><b>2.4</b> Resultados</a></li>
<li class="chapter" data-level="2.5" data-path="2-math-sci.html"><a href="2-math-sci.html#discusión-1"><i class="fa fa-check"></i><b>2.5</b> Discusión</a></li>
<li class="chapter" data-level="2.6" data-path="2-math-sci.html"><a href="2-math-sci.html#conclusión-1"><i class="fa fa-check"></i><b>2.6</b> Conclusión</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-ref-labels.html"><a href="3-ref-labels.html"><i class="fa fa-check"></i><b>3</b> Caracterización molecular</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-ref-labels.html"><a href="3-ref-labels.html#introducción-3"><i class="fa fa-check"></i><b>3.1</b> Introducción</a></li>
<li class="chapter" data-level="3.2" data-path="3-ref-labels.html"><a href="3-ref-labels.html#objetivo-2"><i class="fa fa-check"></i><b>3.2</b> Objetivo</a></li>
<li class="chapter" data-level="3.3" data-path="3-ref-labels.html"><a href="3-ref-labels.html#materiales-y-métodos-2"><i class="fa fa-check"></i><b>3.3</b> Materiales y métodos</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-ref-labels.html"><a href="3-ref-labels.html#análisis-estadístico-1"><i class="fa fa-check"></i><b>3.3.1</b> Análisis estadístico</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-ref-labels.html"><a href="3-ref-labels.html#resultados-2"><i class="fa fa-check"></i><b>3.4</b> Resultados</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-ref-labels.html"><a href="3-ref-labels.html#generación-de-variabilidad-a-partir-del-cruce-biparental-bgsp-00166-x-sp-41255"><i class="fa fa-check"></i><b>3.4.1</b> Generación de variabilidad a partir del cruce biparental BGSP-00166 x SP-41255</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-ref-labels.html"><a href="3-ref-labels.html#discusión-2"><i class="fa fa-check"></i><b>3.5</b> Discusión</a></li>
<li class="chapter" data-level="3.6" data-path="3-ref-labels.html"><a href="3-ref-labels.html#conclusión-2"><i class="fa fa-check"></i><b>3.6</b> Conclusión</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusión-3.html"><a href="conclusión-3.html"><i class="fa fa-check"></i>Conclusión</a></li>
<li class="appendix"><span><b>Apéndice</b></span></li>
<li class="chapter" data-level="A" data-path="A-primer-apéndice.html"><a href="A-primer-apéndice.html"><i class="fa fa-check"></i><b>A</b> Primer Apéndice</a></li>
<li class="chapter" data-level="B" data-path="B-segundo-apéndice.html"><a href="B-segundo-apéndice.html"><i class="fa fa-check"></i><b>B</b> Segundo Apéndice</a></li>
<li class="chapter" data-level="" data-path="referencias.html"><a href="referencias.html"><i class="fa fa-check"></i>Referencias</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Caracterización morfológica, fisiológica y molecular de entradas de algodón (<em>Gossypium hirsutum</em> L.) e identificación de QTL de importancia agronómica</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="referencias" class="section level1 unnumbered hasAnchor">
<h1>Referencias<a href="referencias.html#referencias" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<!--
This manually sets the header for this unnumbered chapter.
-->
<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries. This line may need 
to be removed for styles that don't require the hanging indent.
-->
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0" line-spacing="2">
<div class="csl-entry">
Acquaah, G. (2012). <em>Principles of plant genetics and breeding</em> (2. ed). Wiley-Blackwell.
</div>
<div class="csl-entry">
Ademe, M. S., He, S., Pan, Z., Sun, J., Wang, Q., Qin, H., Liu, J., Liu, H., Yang, J., Xu, D., Yang, J., Ma, Z., Zhang, J., Li, Z., Cai, Z., Zhang, X., Zhang, X., Huang, A., Yi, X., … Du, X. (2017). Association mapping analysis of fiber yield and quality traits in Upland cotton (Gossypium hirsutum L.). <em>Molecular Genetics and Genomics</em>, <em>292</em>(6), 1267-1280. <a href="https://doi.org/10.1007/s00438-017-1346-9">https://doi.org/10.1007/s00438-017-1346-9</a>
</div>
<div class="csl-entry">
An, C., Jenkins, J. N., Wu, J., Guo, Y., &amp; McCarty, J. C. (2010). Use of fiber and fuzz mutants to detect QTL for yield components, seed, and fiber traits of upland cotton. <em>Euphytica</em>, <em>172</em>(1), 21-34. <a href="https://doi.org/10.1007/s10681-009-0009-2">https://doi.org/10.1007/s10681-009-0009-2</a>
</div>
<div class="csl-entry">
ANIDA. (2024). <em>Tipos climáticos en Argentina. Available at: https://ide.ign.gob.ar/portal/apps/MapJournal/index.html?appid=b58d8b8841ef472e962317cac879e87f (Accessed: 24 Apr 2024)</em>.
</div>
<div class="csl-entry">
Bardak, A., Hayat, K., Erdogan, O., Mahmood, Z., Khan, N.-I., Iqbal, M. A., Tekerek, H., &amp; Mehboob-ur-Rahman. (2018). <em>Genetic Mapping in Cotton</em> (Mehboob-Ur-Rahman &amp; Y. Zafar, Eds.). InTech. <a href="https://doi.org/10.5772/intechopen.74513">https://doi.org/10.5772/intechopen.74513</a>
</div>
<div class="csl-entry">
Baytar, A. A., Peynircioğlu, C., Sezener, V., Basal, H., Frary, A., Frary, A., &amp; Doğanlar, S. (2018). Genome-wide association mapping of yield components and drought tolerance-related traits in cotton. <em>Molecular Breeding</em>, <em>38</em>(6), 74. <a href="https://doi.org/10.1007/s11032-018-0831-0">https://doi.org/10.1007/s11032-018-0831-0</a>
</div>
<div class="csl-entry">
Bourland, F. M., Oosterhuis, D. M., &amp; Tugwell, N. P. (1992). Concept for Monitoring the Growth and Development of Cotton Plants Using Main-Stem Node Counts. <em>Journal of Production Agriculture</em>, <em>5</em>(4), 532-538. <a href="https://doi.org/10.2134/jpa1992.0532">https://doi.org/10.2134/jpa1992.0532</a>
</div>
<div class="csl-entry">
De Carvalho, L., Farias, F., Teodoro, L., &amp; Teodoro, P. (2022). Genotype selection for fiber quality traits in cotton in the Brazilian Northeast. <em>Agronomy Journal</em>, <em>114</em>. <a href="https://doi.org/10.1002/agj2.21176">https://doi.org/10.1002/agj2.21176</a>
</div>
<div class="csl-entry">
ICAC. (2023). <em>Situación del Algodón en la Argentina, INFORME CAMPAÑA 2022/2023</em>.
</div>
<div class="csl-entry">
Iqbal, M. A., &amp; Rahman, M.-. (2017). Identification of Marker-Trait Associations for Lint Traits in Cotton. <em>Frontiers in Plant Science</em>, <em>8</em>. <a href="https://doi.org/10.3389/fpls.2017.00086">https://doi.org/10.3389/fpls.2017.00086</a>
</div>
<div class="csl-entry">
Kearsey, M. J., &amp; Pooni, H. S. (1996). <em>The Genetical Analysis of Quantitative Traits</em>. Springer US. <a href="https://doi.org/10.1007/978-1-4899-4441-2">https://doi.org/10.1007/978-1-4899-4441-2</a>
</div>
<div class="csl-entry">
Kerby, T. A., Bourland, F. M., &amp; Hake, K. D. (2010). <em>Physiological Rationales in Plant Monitoring and Mapping</em> (J. McD. Stewart, D. M. Oosterhuis, J. J. Heitholt, &amp; J. R. Mauney, Eds.; pp. 304-317). Springer Netherlands. <a href="https://doi.org/10.1007/978-90-481-3195-2_27">https://doi.org/10.1007/978-90-481-3195-2_27</a>
</div>
<div class="csl-entry">
Kerby, T. A., &amp; Hake, K. D. (1996). Monitoring cotton’s growth. <em>Cotton production manual</em>, 335-355.
</div>
<div class="csl-entry">
Li, C. Q., Dong, N., Fu, Y. Z., Sun, R. R., &amp; Wang, Q. L. (2017). Marker detection and elite allele mining for yield traits in Upland cotton ( <span><em>Gossypium hirsutum</em></span> L.) by association mapping. <em>The Journal of Agricultural Science</em>, <em>155</em>(4), 613-628. <a href="https://doi.org/10.1017/S0021859616000745">https://doi.org/10.1017/S0021859616000745</a>
</div>
<div class="csl-entry">
Lin, Z., He, D., Zhang, X., Nie, Y., Guo, X., Feng, C., &amp; Stewart, J. McD. (2005). Linkage map construction and mapping QTL for cotton fibre quality using SRAP, SSR and RAPD. <em>Plant Breeding</em>, <em>124</em>(2), 180-187. <a href="https://doi.org/10.1111/j.1439-0523.2004.01039.x">https://doi.org/10.1111/j.1439-0523.2004.01039.x</a>
</div>
<div class="csl-entry">
Liu, R., Gong, J., Xiao, X., Zhang, Z., Li, J., Liu, A., Lu, Q., Shang, H., Shi, Y., Ge, Q., Iqbal, M. S., Deng, X., Li, S., Pan, J., Duan, L., Zhang, Q., Jiang, X., Zou, X., Hafeez, A., … Yuan, Y. (2018). GWAS Analysis and QTL Identification of Fiber Quality Traits and Yield Components in Upland Cotton Using Enriched High-Density SNP Markers. <em>Frontiers in Plant Science</em>, <em>9</em>, 1067. <a href="https://doi.org/10.3389/fpls.2018.01067">https://doi.org/10.3389/fpls.2018.01067</a>
</div>
<div class="csl-entry">
Liu, R., Wang, B., Guo, W., Qin, Y., Wang, L., Zhang, Y., &amp; Zhang, T. (2012). Quantitative trait loci mapping for yield and its components by using two immortalized populations of a heterotic hybrid in Gossypium hirsutum L. <em>Molecular Breeding</em>, <em>29</em>(2), 297-311. <a href="https://doi.org/10.1007/s11032-011-9547-0">https://doi.org/10.1007/s11032-011-9547-0</a>
</div>
<div class="csl-entry">
Liu, X., Teng, Z., Wang, J., Wu, T., Zhang, Z., Deng, X., Fang, X., Tan, Z., Ali, I., Liu, D., Zhang, J., Liu, D., Liu, F., &amp; Zhang, Z. (2017). Enriching an intraspecific genetic map and identifying QTL for fiber quality and yield component traits across multiple environments in Upland cotton (Gossypium hirsutum L.). <em>Molecular Genetics and Genomics</em>, <em>292</em>(6), 1281-1306. <a href="https://doi.org/10.1007/s00438-017-1347-8">https://doi.org/10.1007/s00438-017-1347-8</a>
</div>
<div class="csl-entry">
Luo, H. H., Zhang, H. L., Zhang, Y. L., &amp; Zhang, W. F. (2017). Evolution characteristics related to photosynthesis, growth and yield in some old and new cotton cultivars. <em>Photosynthetica</em>, <em>55</em>(2), 301-307. <a href="https://doi.org/10.1007/s11099-016-0223-9">https://doi.org/10.1007/s11099-016-0223-9</a>
</div>
<div class="csl-entry">
Meredith, W. R. (1984). <em>Quantitative Genetics</em> (R. J. Kohel &amp; C. F. Lewis, Eds.; pp. 131-150). American Society of Agronomy, Crop Science Society of America, Soil Science Society of America. <a href="https://doi.org/10.2134/agronmonogr24.c5">https://doi.org/10.2134/agronmonogr24.c5</a>
</div>
<div class="csl-entry">
Naeem, M., Iqbal, M., Sami-Ul-Allah, W. N., &amp; Zahid, W. (2017). QTL Identification for Within-Boll Yield Components of Gossypium hirsutum L. <em>Philippine Agricultural Scientist</em>, <em>100</em>(3).
</div>
<div class="csl-entry">
Nidagundi, J. M., K. Shiva, Revanasiddayya, Sudha Patil, Rashmi S., Hanchinal, S. G., &amp; Ajayakumar, M. Y. (2023). Comparative variability studies for yield and fibre quality traits in F2 generations derived from single and double crosses in cotton (Gossypium hirsutum L). <em>Emergent Life Sciences Research</em>, <em>09</em>(02), 265-275. <a href="https://doi.org/10.31783/elsr.2023.92265275">https://doi.org/10.31783/elsr.2023.92265275</a>
</div>
<div class="csl-entry">
Paterson, A. H., Brubaker, C. L., &amp; Wendel, J. F. (1993). A rapid method for extraction of cotton (Gossypium spp.) genomic DNA suitable for RFLP or PCR analysis. <em>Plant Molecular Biology Reporter</em>, <em>11</em>(2), 122-127. <a href="https://doi.org/10.1007/BF02670470">https://doi.org/10.1007/BF02670470</a>
</div>
<div class="csl-entry">
Paytas, M. J., &amp; Ploschuk, E. L. (2013). <em>Cultivos industriales-Capítulo Algodón</em> (E. De La Fuente, A. Gil, &amp; A. Kantolil, Eds.). Facultad de Agronomía Universidad de Buenos Aires.
</div>
<div class="csl-entry">
Pettigrew, W. T., Meredith Jr, W., &amp; Zeng, L. (2013). Response of obsolete and modern cotton genotypes to varying plant densities. <em>Journal of Cotton Science</em>, <em>17</em>(4), 253-262.
</div>
<div class="csl-entry">
Qin, H., Chen, M., Yi, X., Bie, S., Zhang, C., Zhang, Y., Lan, J., Meng, Y., Yuan, Y., &amp; Jiao, C. (2015). Identification of Associated SSR Markers for Yield Component and Fiber Quality Traits Based on Frame Map and Upland Cotton Collections. <em>PLOS ONE</em>, <em>10</em>(1), e0118073. <a href="https://doi.org/10.1371/journal.pone.0118073">https://doi.org/10.1371/journal.pone.0118073</a>
</div>
<div class="csl-entry">
Ribeiro, L. P., Carvalho, L. P. de, Farias, F. J. C., Rodrigues, J. I. da S., Teodoro, P. E., &amp; Bhering, L. L. (2017). Genetic gains in agronomic and technological traits of elite cotton genotypes. <em>Bragantia</em>, <em>77</em>, 466-475. <a href="https://doi.org/10.1590/1678-4499.2017329">https://doi.org/10.1590/1678-4499.2017329</a>
</div>
<div class="csl-entry">
Rodriguez, G. R., Pratta, G. R., Zorzoli, R., &amp; Picardi, L. A. (2005). Transgressive segregation for fruit quality traits in a cross between exotic and mutant genotypes of <span><em>Lycopersicon</em></span>. <em>New Zealand Journal of Crop and Horticultural Science</em>, <em>33</em>(4), 373-379. <a href="https://doi.org/10.1080/01140671.2005.9514373">https://doi.org/10.1080/01140671.2005.9514373</a>
</div>
<div class="csl-entry">
Royo, O. M., Poisson, J., Bonacic, I., Montenegro, A., Ibalo, S., Mazza, S., &amp; Giménez, L. (2007). <em>Direction of cotton breeding in Argentina.</em>
</div>
<div class="csl-entry">
Scarpin, G. J., Dileo, P. N., Winkler, H. M., Cereijo, A. E., Lorenzini, F. G., Muchut, R. J., Roeschlin, R. A., Mattalia, P., Acuña, C., &amp; Paytas, M. (2023). Genetic progress in seed and fiber quality traits of cotton in Argentina. <em>Field Crops Research</em>, <em>302</em>, 109106. <a href="https://doi.org/10.1016/j.fcr.2023.109106">https://doi.org/10.1016/j.fcr.2023.109106</a>
</div>
<div class="csl-entry">
Scarpin, G. J., Dileo, P. N., Winkler, H. M., Cereijo, A. E., Lorenzini, F. G., Roeschlin, R. A., Muchut, R. J., Acuña, C., &amp; Paytas, M. (2022). Genetic progress in cotton lint and yield components in Argentina. <em>Field Crops Research</em>, <em>275</em>, 108322. <a href="https://doi.org/10.1016/j.fcr.2021.108322">https://doi.org/10.1016/j.fcr.2021.108322</a>
</div>
<div class="csl-entry">
Shen, X., Guo, W., Lu, Q., Zhu, X., Yuan, Y., &amp; Zhang, T. (2007). Genetic mapping of quantitative trait loci for fiber quality and yield trait by RIL approach in Upland cotton. <em>Euphytica</em>, <em>155</em>(3), 371-380. <a href="https://doi.org/10.1007/s10681-006-9338-6">https://doi.org/10.1007/s10681-006-9338-6</a>
</div>
<div class="csl-entry">
Shi, Y., Li, W., Li, A., Ge, R., Zhang, B., Li, J., Liu, G., Li, J., Liu, A., &amp; Shang, H. (2015). Constructing a high<span>-</span>density linkage map for Gossypium hirsutum× Gossypium barbadense and identifying QTLs for lint percentage. <em>Journal of integrative plant biology</em>, <em>57</em>(5), 450-467.
</div>
<div class="csl-entry">
Su, J., Fan, S., Li, L., Wei, H., Wang, C., Wang, H., Song, M., Zhang, C., Gu, L., Zhao, S., Mao, G., Wang, C., Pang, C., &amp; Yu, S. (2016). Detection of Favorable QTL Alleles and Candidate Genes for Lint Percentage by GWAS in Chinese Upland Cotton. <em>Frontiers in Plant Science</em>, <em>7</em>. <a href="https://doi.org/10.3389/fpls.2016.01576">https://doi.org/10.3389/fpls.2016.01576</a>
</div>
<div class="csl-entry">
Tang, B., Jenkins, J. N., Watson, C. E., McCarty, J. C., &amp; Creech, R. G. (1996). Evaluation of genetic variances, heritabilities, and correlations for yield and fiber traits among cotton F2 hybrid populations. <em>Euphytica</em>, <em>91</em>(3), 315-322. <a href="https://doi.org/10.1007/BF00033093">https://doi.org/10.1007/BF00033093</a>
</div>
<div class="csl-entry">
Wang, B., Guo, W., Zhu, X., Wu, Y., Huang, N., &amp; Zhang, T. (2007). QTL Mapping of Yield and Yield Components for Elite Hybrid Derived-RILs in Upland Cotton. <em>Journal of Genetics and Genomics</em>, <em>34</em>(1), 35-45. <a href="https://doi.org/10.1016/S1673-8527(07)60005-8">https://doi.org/10.1016/S1673-8527(07)60005-8</a>
</div>
<div class="csl-entry">
Wang, H., Huang, C., Guo, H., Li, X., Zhao, W., Dai, B., Yan, Z., &amp; Lin, Z. (2015). QTL Mapping for Fiber and Yield Traits in Upland Cotton under Multiple Environments. <em>PLOS ONE</em>, <em>10</em>(6), e0130742. <a href="https://doi.org/10.1371/journal.pone.0130742">https://doi.org/10.1371/journal.pone.0130742</a>
</div>
<div class="csl-entry">
Wang, M., Li, C., &amp; Wang, Q. (2014). Quantitative trait loci mapping and genetic dissection for lint percentage in upland cotton (Gossypium hirsutum). <em>Journal of Genetics</em>, <em>93</em>(2), 371-378. <a href="https://doi.org/10.1007/s12041-014-0385-9">https://doi.org/10.1007/s12041-014-0385-9</a>
</div>
<div class="csl-entry">
Wells, R., &amp; Meredith, W. (1984). Comparative Growth of Obsolete and Modern Cotton Cultivars. III. Relationship of Yield to Observed Growth Characteristics1. <em>Crop Science - CROP SCI</em>, <em>24</em>. <a href="https://doi.org/10.2135/cropsci1984.0011183X002400050010x">https://doi.org/10.2135/cropsci1984.0011183X002400050010x</a>
</div>
<div class="csl-entry">
Worley, S., Culp, T., &amp; Harrell, D. (1974). The relative contributions of yield components to lint yield of upland cotton, Gossypium hirsutum L. <em>Euphytica</em>, <em>23</em>(2), 399-403.
</div>
<div class="csl-entry">
Xia, Z., Zhang, X., Liu, Y., Jia, Z., Zhao, H., Li, C., &amp; Wang, Q. (2014). Major Gene Identification and Quantitative Trait Locus Mapping for Yield-Related Traits in Upland Cotton (Gossypium hirsutum L.). <em>Journal of Integrative Agriculture</em>, <em>13</em>(2), 299-309. <a href="https://doi.org/10.1016/S2095-3119(13)60508-0">https://doi.org/10.1016/S2095-3119(13)60508-0</a>
</div>
<div class="csl-entry">
Zhang, J., &amp; Stewart, J. M. (2000). Economical and Rapid Method for Extracting Cotton Genomic DNA. <em>J Cotton Sci</em>, <em>4</em>(3).
</div>
<div class="csl-entry">
Zhang, S., Feng, L., Xing, L., Yang, B., Gao, X., Zhu, X., Zhang, T., &amp; Zhou, B. (2016). New QTL s for lint percentage and boll weight mined in introgression lines from two feral landraces into Gossypium hirsutum acc TM<span>-</span>1. <em>Plant Breeding</em>, <em>135</em>(1), 90-101.
</div>
<div class="csl-entry">
Zhang, T., Yuan, Y., Yu, J., Guo, W., &amp; Kohel, R. J. (2003). Molecular tagging of a major QTL for fiber strength in Upland cotton and its marker-assisted selection. <em>Theoretical and Applied Genetics</em>, <em>106</em>(2), 262-268. <a href="https://doi.org/10.1007/s00122-002-1101-3">https://doi.org/10.1007/s00122-002-1101-3</a>
</div>
<div class="csl-entry">
Zhang, Z.-S., Xiao, Y.-H., Luo, M., Li, X.-B., Luo, X.-Y., Hou, L., Li, D.-M., &amp; Pei, Y. (2005). Construction of a genetic linkage map and QTL analysis of fiber-related traits in upland cotton (Gossypium hirsutum L.). <em>Euphytica</em>, <em>144</em>(1-2), 91-99. <a href="https://doi.org/10.1007/s10681-005-4629-x">https://doi.org/10.1007/s10681-005-4629-x</a>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="B-segundo-apéndice.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["Tesis_Dileo.pdf", "PDF"], ["Tesis_Dileo.epub", "EPUB"], ["Tesis_Dileo.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
